parser.y

```diff
@@ -30,6 +30,7 @@ rule
 
   bison_declaration: grammar_declaration
                    | rule_declaration
+                   | inline_declarations
```

Comment:
I guess `%inline` is an optional attribute for `%rule` then its production rules are declared as a part of `rule_declaration` like:

```
rule_declaration: "%rule" inline_opt IDENTIFIER "(" rule_args ")" ":" rule_rhs_list`

inline_opt: # empty
          | "%inline"           
```

Reply:
Indeed it is. I changed `%inline` to an optional attribute of `%rule` https://github.com/ruby/lrama/pull/369/commits/0cf305a2d46ec98a804943ee0d259e23762d7258